packages/components/psammead-script-switch/src/index.jsx

```diff
@@ -24,41 +16,43 @@ const StyledLink = styled.a`
   color: ${C_WHITE};
   text-align: center;
   text-decoration: none;
+  padding: 0 1rem;
+  height: 3rem;
 
   &:hover {
     border-width: 0.25rem;
     line-height: 2.675rem;
+    padding: 0 0.8rem;
   }
   &:focus {
     background-color: ${C_WHITE};
     color: ${C_EBON};
   }
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```

Comment:
I believe this media query should be removed.

Reply:
Below is a `max-width` this one is a `min-width`. I have added this media query because `${({ script }) => script && getPica(script)}` will return a media query definition with `line-height` defined. And this  media query is used to override the `line-height` from `getPica`